src/main/java/appeng/me/storage/BasicCellInventoryHandler.java

```diff
@@ -88,7 +88,11 @@ public AbstractCellInventoryHandler( final IMEInventory c, final IStorageChannel
 				final ItemStack is = config.getStackInSlot( x );
 				if( !is.isEmpty() )
 				{
-					priorityList.add( createConfigStackFromItem( is ) );
+					final T configItem = channel.createStack( is );
+					if( configItem != null )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS134)